tests/NuGet.Services.AzureSearch.Tests/HijackDocumentBuilderFacts.cs

```diff
@@ -159,11 +159,11 @@ public async Task SetsExpectedProperties()
       ],
       ""title"": ""Windows Azure Storage"",
       ""tokenizedPackageId"": ""WindowsAzure.Storage"",
+      ""lastCommitTimestamp"": null,
```

Comment:
The only times we should be setting these values to null is when the record is initialized from DB.